### ISSUE TITLE
uploading big file

### ISSUE THREAD

**Issue by bgbruno** [getsentry/sentry-symfony#268]
Hello Sentry,

after client upload on page 20MB file - sentry slow down process to render it +3seconds.

**Q how can I exclude sentry from element or url ?**

Bruno

**Comment by Jean85** [comment]
How are you uploading this element? If it's in a POST body, you could exclude that with the `max_request_body_size => none` option.

**Comment by Jean85** [comment]
Closing as stale. Further information could be taken [from the docs](https://docs.sentry.io/platforms/php/).

**Comment by Gabb1995** [comment]
Is it possible to exclude a specific action?

In my case I am also uploading a 20MB via POST request base64encoded and its always running out of memory here:

`RequestIntegration->captureRequestBody()`

I have set the max_request_body_size set to small and when it does the following if statement:

```
        if (
            'none' === $maxRequestBodySize ||
            ('small' === $maxRequestBodySize && $requestBody->getSize() > self::REQUEST_BODY_SMALL_MAX_CONTENT_LENGTH) ||
            ('medium' === $maxRequestBodySize && $requestBody->getSize() > self::REQUEST_BODY_MEDIUM_MAX_CONTENT_LENGTH)
        ) {
            return null;
        }
```

`$requestBody->getSize()` returns null so it never goes in this. I looked further in and found that in my case the $requestBody is of instance: Zend\Diactoros\PhpInputStream, and in here getSize calls fstat to get the size and it returns 0 so the function getSize is returning null.
